CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1040,8 +1040,29 @@ local function CreateSelector(selectionStringTable, startPosition)
 			local isSelected = (i == index)
 
 			if not selectionLabel:IsDescendantOf(game) then
-				this.CurrentIndex = i
-				indexChangedEvent:Fire(index)
+				for i,selectionLabel in pairs(this.Selections) do	
```

Comment:
Yes, when `:SetSelectionIndex()` is fired from GameSettings the selectionLabel aren't considered a descendant of game. And throws the error 

> Can only tween objects in the workspace

And that part of the code is only ran when Fullscreen is toggled with F11 not the slider.